common/lib/common-utils/src/performanceNode.ts

```diff
@@ -3,6 +3,9 @@
  * Licensed under the MIT License.
  */
 
+import { performance, PerformanceObserver } from "perf_hooks";
+export { performance, PerformanceObserver };
+
 export function performanceNow() {
```

Comment:
if performance is being exported, this should just be removed and the consumers updated like in trace.ts

Reply:
What do you mean? How will projects outside of common-utils reference `performance`?

Reply:
```
import { performance } from "@fluidframework/common-utils";
performance.now()
```
If `performance` is getting exported from here anyway, we shouldn't also export the `performanceNow` wrapper because it would duplicate the functionality of the `performance` export (unless the node version can't be used in the same way?)

Reply:
Oh yes definitely (sorry I misunderstood which line you wanted me to remove).

I only left it in for backwards compatibility.. do we need to keep common +/-1 version compatible?

Reply:
I'm not sure what happens when we dynamically load container code from the loader, does it bring in its own copy of common-utils?

Reply:
Got it, thanks. So I'll remove the back-compat function.